gateway-service-impl/src/test/resources/configs/gateway-service/application.conf

```diff
@@ -17,6 +17,9 @@ attributes.service.config = {
   host = localhost
   port = 9012
 }
+query.executor.config = {
```

Comment:
Can we add the helm too?

Reply:
can be added in follow-up.

Reply:
the helm config map template takes in a blob for application conf, so no changes are needed there (I could have overridden this default, but that didn't seem necessary):

https://github.com/hypertrace/gateway-service/blob/main/helm/values.yaml#L67-L76